.cirrus.yml

```diff
@@ -743,13 +741,6 @@ artifacts_task:
         - unzip gosrc.zip
         - cd $CIRRUS_WORKING_DIR
         - mv /tmp/alt/*.tar.gz ./
-        # Windows MSI filename has version number
-        - mkdir -p /tmp/win
-        - cd /tmp/win
-        - $CURL/Windows%20Cross/gosrc.zip
-        - unzip gosrc.zip
-        - cd $CIRRUS_WORKING_DIR
-        - mv /tmp/win/podman-remote*.zip /tmp/win/*.msi ./
```

Comment:
This is confusing to me WRT the note in the commit message: "Remove windows installer build task".